subprojects/launcher/src/integTest/groovy/org/gradle/launcher/CommandLineIntegrationSpec.groovy

```diff
@@ -41,7 +41,7 @@ class CommandLineIntegrationSpec extends AbstractIntegrationSpec {
         fails "help"
 
         and:
-        errorOutput.trim().readLines()[0] == "Argument value '$value' given for --max-workers option is invalid (must be a positive, non-zero, integer)"
+        failure.assertHasDescription "Argument value '$value' given for system property org.gradle.workers.max or --max-workers option is invalid (must be a positive, non-zero, integer)"
```

Comment:
this isn't great. We know where the value came from, we shouldn't make the user go hunting for it.